install/csm_installation_failure.md

```diff
@@ -0,0 +1,91 @@
+# CSM Services Install Fails Because of Missing Secret
+
+When running the install script in the [Install CSM Services](install_csm_services.md) procedure, it may fail due to a
+timing-related issue. This page documents how to determine if this problem was the cause of an install script failure,
+and the appropriate remediation steps to take if it is encountered.
+
+## How to determine if an install hit this issue
+
+1. Verify that the installation script output contains an error about a missing secret.
+
+    Example snippet from the install script output:
+
+    ```text
+      ERROR   Step: Set Management NCNs to use Unbound --- Checking Precondition
+    + Getting admin-client-auth secret
+    Error from server (NotFound): secrets "admin-client-auth" not found
+    + Obtaining access token
+    curl: (22) The requested URL returned error: 404
+    + Querying SLS
+    curl: (22) The requested URL returned error: 503
+
+    Check the doc below for troubleshooting:
+          If any management NCNs are missing from the output, take corrective action
+          before proceeding.
+
+     INFO  Failed Pipeline/Step id: f0cd9574240989eb04118d308f26b3ea
+    exit status 22
+    ```
+
+    If the above `secrets "admin-client-auth" not found` error is observed, then proceed to the next step.
+
+## Validate Keycloak Setup Is Running
```

Comment:
It seems like this still falls under the heading of determining if the install hit this issue, so we don't need a new header.
```suggestion
```